### DISCUSSION TITLE
How to highlight Markdown inside HTML without a blank line?

### DISCUSSION THREAD

**Discussion by LeaVerou** [markedjs/marked#2317] [Q&A]
I noticed that marked does not support anything like the `markdown="1"` attribute of other Markdown parsers. (see #488) and that after #1135, a blank line is required to switch to Markdown mode again.
While this works for things like `<div>`, it does not work for highlighting Markdown inside e.g. `<p>`.

It seems I can use the `html()` token in Renderer or Tokenizer to fix this, but I'm not sufficiently familiar with Marked's internals. Has anyone else figured this out?

**Comment by UziTech** [comment]
https://github.com/markedjs/marked/issues/1907#issuecomment-855525022
